lib/oli/authoring/course.ex

```diff
@@ -47,7 +47,7 @@ defmodule Oli.Authoring.Course do
     case author do
       # Admin authors have access to every project
       %{system_role_id: ^admin_role_id} -> Repo.all(Project)
-      _ -> Repo.preload(author, [:projects]).projects
+      _ -> Repo.preload(author, [:projects]).projects |> Enum.filter(fn p -> p.status === :active end)
```

Comment:
The filtering by status should be removed here, it changes the purpose of this function.  Instead, move the filtering by status to a higher level piece of code that is interested in active projects only.

Reply:
move filtering to higher-level code where list of projects is rendered